test/handler-tests.js

```diff
@@ -1469,4 +1471,35 @@ describe('handler', function() {
         });
     });
   });
+
+  describe('Upsert', function() {
+    const testDoc = { field: uuid.v4() };
```

Comment:
Please, wrap this test in something like
```js
it('should use "update" action', function() {
  // ...
});
```

Reply:
Will do! 👍